frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/createCommentBox/createCommentBox.component.tsx

```diff
@@ -0,0 +1,246 @@
+			error: imageIsTooBig(file),
+			id: uuid(),
+		})));
+		setImagesToUpload(imagesToUpload.concat(images));
+	};
+
+	const uploadImages = async () => {
+		const files = await uploadFile(getSupportedImageExtensions(), true) as File[];
+		await uploadFiles(files);
+	};
+
+	const uploadScreenshot = async () => {
+		const image = await ViewerService.getScreenshot() as unknown as string;
+		const imageToUpload: ImageToUpload = {
+			src: image,
+			id: uuid(),
+		};
+		setImagesToUpload(imagesToUpload.concat(imageToUpload));
+	};
+
+	const deleteImage = (idToDelete) => {
+		setImagesToUpload(imagesToUpload.filter(({ id }) => id !== idToDelete));
+	};
+
+	const openImagesModal = (imageIndex) => {
+		DialogsActionsDispatchers.open('images', {
+			images: imagesToUpload.map(({ src }) => src),
+			displayImageIndex: imageIndex,
+		});
+	};
+
+	const handleDragLeave = ({ relatedTarget }) => {
+		if (containerRef.current.contains(relatedTarget) || !relatedTarget.parentElement) return;
+		setIsDraggingFile(false);
+	};
+
+	useEffect(() => { resetCommentBox(); }, [ticketId]);
+
+	useEffect(() => {
+		if (commentReply?.message || commentReply?.images?.length) {
+			inputRef.current.focus();
+		}
+	}, [commentReply?._id]);
+
+	return (
+		<Container
+			onDragEnter={() => setIsDraggingFile(true)}
+			onDragLeave={handleDragLeave}
+			onDrop={() => setIsDraggingFile(false)}
+			ref={containerRef}
+		>
+			{commentReply && (
+				<CommentReplyContainer>
+					<CommentReply {...replyMetadata} shortMessage />
+					<DeleteButton onClick={() => setCommentReply(null)}>
+						<DeleteIcon />
+					</DeleteButton>
+				</CommentReplyContainer>
+			)}
+			<MessageInput
+				name="message"
+				placeholder={formatMessage({
+					id: 'customTicket.comments.leaveAComment',
+					defaultMessage: 'Leave a comment',
+				})}
+				control={control}
+				inputProps={{
+					maxLength: Math.max(MAX_MESSAGE_LENGTH - commentReplyLength, 0),
+				}}
+				inputRef={inputRef}
+			/>
+			<DragAndDrop accept={getSupportedImageExtensions()} onDrop={uploadFiles} hidden={!isDraggingFile}>
+				<FormattedMessage
+					id="customTicket.comments.dropFiles"
+					defaultMessage="Drop your files here"
+				/>
+			</DragAndDrop>
+			{imagesToUpload.length > 0 && (
+				<ScrollArea autoHeightMax={100} autoHeight autoHide>
+					<Images>
+						{imagesToUpload.map(({ src, id, error }, index) => (
+							<ImageContainer key={id}>
+								<Image src={src} $error={error} onClick={() => openImagesModal(index)} />
+								<DeleteButton onClick={() => deleteImage(id)} error={error}>
+									<DeleteIcon />
+								</DeleteButton>
+							</ImageContainer>
+						))}
+					</Images>
+				</ScrollArea>
+			)}
+			{erroredImages.length > 0 && erroredImages.map(({ name }) => (
+				<ErroredImageMessage>
+					<strong>{name} </strong>
+					<FormattedMessage
+						id="customTicket.comments.images.error"
+						defaultMessage="is too big. 1GB limit."
```

Comment:
Should the filesize here be taken from the config?

Reply:
And is it not possible for other errors to occur?

Reply:
Error should be known in order to be displayed properly. The only 2 errors that I can think of are:
1. The file is too big
2. The file is corrupted: the extension asserts to be an allowed one, but the in the backend a check is performed and that image is actually not valid (e.g. webp)

The second error triggers a modal explaining the issue, but it not possible to detect what file is responsible for the error since the error response does not contain any information about it.

Regarding the first error, for `editProfileAvatar` (`src\v5\validation\userSchemes\validators.ts`) the size is hardcoded. There is an helper function that maps the size value to a string (`formatInfoUnit`), but the output is "104.9 MB". If @sanmont3drepo is ok with that, I can use that function instead